definitions/procedures/restore/extract_files.rb

```diff
@@ -71,6 +71,10 @@ def extract_pgsql_data(backup)
         :transform => any_database.restore_transform
       )
       feature(:tar).run(pgsql_data_tar)
+      del_data_dir_param if el?
```

Comment:
I'm not 100% sure this is limited to EL. It's the puppetlabs-postgresql module which applies this and if they used our installer it's probably present. However, I'm unsure if that's a problem since I'm too unfamiliar with foreman-maintain and where this code is used.